uint64[] nodes
string mode

---
bool result